src/coreclr/src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -445,6 +445,8 @@ public static object GetUniqueObjectForIUnknown(IntPtr unknown)
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern bool IsComObject(object o);
 
+        internal static bool IsComSupported => true;
```

Comment:
This needs to read the runtime property to correctly react to the feature switch.